screens/Scan/ScanScreen.tsx

```diff
@@ -151,13 +154,28 @@ export const ScanScreen: React.FC = () => {
   function displayStorageLimitReachedError(): React.ReactNode {
     return (
       !controller.isEmpty && (
-        <ErrorMessageOverlay
+        <MessageOverlay
           isVisible={
             controller.isMinimumStorageRequiredForAuditEntryLimitReached
           }
           translationPath={'ScanScreen'}
           error="errors.storageLimitReached"
-          onDismiss={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
+          onBackdropPress={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
+        />
+      )
+    );
+  }
+
+  function invalidQR(): React.ReactNode {
```

Comment:
functional component name usually starts with Capital Letter. can we refactor this as well.